libr/debug/p/native/w32.c

```diff
@@ -833,7 +617,7 @@ static RDebugPid *build_debug_pid(PROCESSENTRY32 *pe) {
 	return ret;
 }
 
-RList *w32_pids (int pid, RList *list) {
+RList *w32_pids(int pid, RList *list) {
```

Comment:
why get a list to return it? just create it inside this funtion and remove the arg

Reply:
show_all_pids should be bool

Reply:
> why get a list to return it? just create it inside this funtion and remove the arg

I checked and the equivalent functions for the other platforms also ask for a list. I wont change them because it is out of scope for this pr